apps-rendering/src/components/Headline/GalleryHeadline.tsx

```diff
@@ -43,6 +49,10 @@ const backgroundStyles = css`
 		${grid.between('centre-column-start', 'viewport-end')}
 		margin-left: calc(${grid.columnGap} * -1/2);
 	}
+
+	${darkModeCss`
+		background-color: ${neutral[10]};
```

Comment:
One for the palette?

```suggestion
		background-color: ${background.headlineDark(format)};
```